### PR TITLE
[Merged by Bors] - Add printing of kibana ip

### PR DESCRIPTION
**Pull request by valar999**
## Motivation

Print Kibana IP in systemtests.

### CODE REVIEW

File: tests/utils.py [hunk 1]
```diff
@@ -251,7 +255,53 @@ def wait_for_minimal_elk_cluster_ready(namespace, es_ss_name=ES_SS_NAME, logstas
...
         raise Exception(f"logstash took over than {ls_timeout} to start")
 
-    return logstash_sleep_time + es_sleep_time
+    kb_timeout = 240
+    try:
+        print("waiting for kibana to be ready")
+        kibana_sleep_time = statefulset.wait_to_statefulset_to_be_ready(kibana_ss_name, namespace, time_out=ls_timeout)
+    except Exception as e:
+        print(f"got an exception while waiting for keban to be ready: {e}")
+        raise Exception(f"keban took over than {kb_timeout} to start")
```
Comment (noamnelke) [inline tests/utils.py]:
```suggestion
        print(f"got an exception while waiting for kibana to be ready: {e}")
        raise Exception(f"kibana took more than {kb_timeout} to start")
```

File: tests/utils.py [hunk 2]
```diff
@@ -233,7 +235,9 @@ def wait_genesis(genesis_time, genesis_delta):
         time.sleep(delta_from_genesis)
 
 
-def wait_for_minimal_elk_cluster_ready(namespace, es_ss_name=ES_SS_NAME, logstash_ss_name=LOGSTASH_SS_NAME):
+def wait_for_minimal_elk_cluster_ready(namespace, es_ss_name=ES_SS_NAME,
+                                                  logstash_ss_name=LOGSTASH_SS_NAME,
+                                                  kibana_ss_name=KIBANA_SS_NAME):
```
Comment (AmitShaul) [inline tests/utils.py]: ss stands for statefulset, Kibana is a deployment it should be named KIBANA_DEP_NAME

Comment (AmitShaul) [reply tests/utils.py]: same with kibana_ss_name

**Review by AmitShaul — APPROVED** [review]

**Comment by valar999** [comment]
bors merge

**Comment by valar999** [comment]
bors merge

**Comment by valar999** [comment]
bors merge

**Comment by valar999** [comment]
bors merge

**Comment by valar999** [comment]
bors merge

**Comment by valar999** [comment]
bors merge

**Comment by valar999** [comment]
bors merge

**Comment by valar999** [comment]
bors try

**Comment by valar999** [comment]
bors try

**Comment by valar999** [comment]
bors try

**Comment by valar999** [comment]
bors try

**Comment by valar999** [comment]
bors try

**Comment by valar999** [comment]
bors try

**Comment by valar999** [comment]
bors try

**Comment by valar999** [comment]
bors try

**Comment by valar999** [comment]
bors try

**Comment by valar999** [comment]
bors merge

**Pull request closed without merging**